docs/source/en/task_summary.mdx

```diff
@@ -34,8 +34,12 @@ Audio classification is a task that labels audio data from a predefined set of c
 ```py
 >>> from transformers import pipeline
 
->>> classifier = pipeline(task="audio-classification")
->>> classifier("path/to/audio/file.mp3")
+>>> classifier = pipeline(task="audio-classification", model="superb/hubert-base-superb-er")
+>>> classifier("https://huggingface.co/datasets/Narsil/asr_dummy/resolve/main/mlk.flac")
+[{'score': 0.453214168548584, 'label': 'hap'},
+ {'score': 0.3622128367424011, 'label': 'sad'},
+ {'score': 0.09430035948753357, 'label': 'neu'},
+ {'score': 0.09027273952960968, 'label': 'ang'}]
```

Comment:
Scores should only be printed with 3 or 4 digits, or the tests can't be reproduced on different hardware.